driver-core/src/main/java/com/datastax/driver/core/Statement.java

```diff
@@ -224,6 +224,9 @@ public abstract ByteBuffer getRoutingKey(
    * method is thus only useful in case you want to punctually override the default policy for this
    * request.
    *
+   * <p>Note that unlike other configuration, when this statement is prepared {@link
+   * BoundStatement}s created off of {@link PreparedStatement} do not inherit this configuration.
+   *
```

Comment:
It looks like the retry policy is actually propagated: https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/AbstractSession.java#L131

Reply:
Ah, I missed that somehow, good catch!